distributed/client.py

```diff
@@ -27,6 +27,7 @@
 from queue import Queue as pyQueue
 from typing import Any, ClassVar, Coroutine, Literal, Sequence, TypedDict
 
+import fsspec as fsspec
```

Comment:
```suggestion
import fsspec
```

Reply:
`fsspec` also isn't a required dependency for `distributed`. At the moment it will always be installed because `distributed` depends on `dask` and `dask` depends on `fsspec`, but that could change in the future. Probably best to gate this import inside the `__aexit__` call below where `fsspec.open` is called. That would also avoid import-time slowdowns in cases where `performance_report`s aren't used